### ISSUE TITLE
Improve 01526 message when sampleLocationsPerPixel is 0

### ISSUE THREAD

**Issue by ziga-lunarg** [KhronosGroup/Vulkan-ValidationLayers#8680]
If `VkSampleLocationsInfoEXT::sampleLocationsPerPixel` is 0, the validation layers message will say `Unhandled VkSampleCountFlagBits`

Full message is:
> vkCmdSetSampleLocationsEXT(): pSampleLocationsInfo->sampleLocationsPerPixel is Unhandled VkSampleCountFlagBits, but VkPhysicalDeviceSampleLocationsPropertiesEXT::sampleLocationSampleCounts is VK_SAMPLE_COUNT_1_BIT|VK_SAMPLE_COUNT_2_BIT|VK_SAMPLE_COUNT_4_BIT|VK_SAMPLE_COUNT_8_BIT|VK_SAMPLE_COUNT_16_BIT.

**Comment by spencer-lunarg** [comment]
so `zero` is not a valid `VkSampleCountFlagBits` so what would you prefer?

> pSampleLocationsInfo->sampleLocationsPerPixel is Unhandled VkSampleCountFlagBits (0x0bad), but

**Comment by ziga-lunarg** [comment]
Yes, I would prefer to see the invalid value, rather than `Unhandled VkSampleCountFlagBits`.

**Comment by spencer-lunarg** [comment]
ok, that is fair, will get a fix up soon-ish

**Comment by spencer-lunarg** [comment]
so this is something we need to fix in VUL then, `vk_enum_string_helper.h` has `string_VkSampleCountFlagBits` which is printing `Unhandled VkSampleCountFlagBits` and would need to where we would print the value when using an `Unhandled` value
